SQL/0000-00-00-schema.sql

```diff
@@ -2355,3 +2355,14 @@ CREATE TABLE `issues_watching` (
   KEY `fk_issues_watching_2` (`issueID`),
   CONSTRAINT `fk_issues_watching_1` FOREIGN KEY (`userID`) REFERENCES `users` (`UserID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `monitoring`;
+CREATE TABLE `monitoring` (
+  `CandID` int(6) NOT NULL,
+  `visit_label` varchar(255) DEFAULT NULL,
```

Comment:
First letter should be Upper-case to be consistent with the rest of the table-fields. i.e Session.Visit_label